src/Components/Input.tsx

```diff
@@ -75,51 +91,32 @@ class Input extends React.Component<InputProps, InputState> {
     }
   }
 
-  render() {
-    if (this.props.leftView && this.props.rightView) {
-      const { className, ref, ...newProps } = this.props
+  onChange(e: React.FocusEvent<HTMLInputElement>) {
```

Comment:
perhaps `onChange = () => {...}` here to bind the callback? idk if it's possible to overuse this but this is one pattern I'd like to see us consistently use in reaction.

Reply:
Yeah, unfortunately it's incompatible with the type (`React.FocusEvent<HTMLInputElement>`) but I think that's ok